### PR TITLE
Multiple small fixes

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: packages/openapi-to-graphql/src/schema_builder.ts [hunk 1]
```diff
@@ -870,20 +870,48 @@ export function getArgs({
...
+          // Check header option
+          if (
+            typeof data.options.headers === 'object' &&
+            parameter.name in data.options.headers
+          ) {
+            continue
+          }
+
+          // Check requestOptions option
+          if (
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/schema_builder.ts]: Now it checks the `requestOptions` as well

File: packages/openapi-to-graphql/src/schema_builder.ts [hunk 2]
```diff
@@ -870,20 +870,48 @@ export function getArgs({
...
-        parameter.name in data.options.headers
-      ) {
-        continue
-      }
-      if (
-        typeof data.options.qs === 'object' &&
-        parameter.name in data.options.qs
-      ) {
-        continue
+      switch (parameter.in) {
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/schema_builder.ts]: Before, we were checking both the `qs` and the `headers` options for fulfillment. This is bad. They should have a limited scope.

File: packages/openapi-to-graphql/test/example_api.test.ts [hunk 3]
```diff
@@ -1830,3 +1830,171 @@ test('Option selectQueryOrMutationField', () => {
 
   return Promise.all([promise, promise2])
 })
+
+test('Header arguments are not created when they are provided through headers option', () => {
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/test/example_api.test.ts]: Added multiple clear tests.

**Comment by ErikWittern** [comment]
@Alan-Cha Nice work, looks good to me!

**Pull request merged**